comparison_blogpost.txt

```diff
@@ -0,0 +1,59 @@
+Summary
```

Comment:
I made some comments you can start working on. The first blog looks pretty good to me. 
I need to think on teh comparison blog a bit more to give better feedback but looks very good start for me.